tests/e2e-pw/tests/cart-block/cart-block.merchant.block_theme.spec.ts

```diff
@@ -0,0 +1,194 @@
+/**
```

Comment:
I left a comment earlier but I think it got lost once you pushed some changes. Anyway I was suggesting a file structure of `tests/cart-checkout/testname.spec.ts`. Mainly because we're dealing with e2e tests here, some of which will span both cart and checkout blocks. So it makes sense to group them together here rather than separate directories by blocks.